src/frontend/hooks/useLastSavedLocation.ts

```diff
@@ -0,0 +1,10 @@
+import {useQuery} from '@tanstack/react-query';
+import {getLastKnownPositionAsync} from 'expo-location';
+
+export const useLastSavedLocation = () => {
+  return useQuery({
+    queryKey: ['lastLocation'],
+    queryFn: async () =>
+      getLastKnownPositionAsync().catch(err => console.log(err)),
```

Comment:
I don't think to. There are 2 use cases: have a coordinate to show when setting the coordinate format and for showing a location on the map, while the actual location loads.

In both cases, the info doesn't actually matter (aka. were not saving the information). We use it because it is better than having undefined data. So setting a maxAge or requiredAccuracy, means there is a greater chance the information is null, which defeats the purpose of using this hook.